site/en/docs/extensions/migrating/known-issues/index.md

```diff
@@ -15,69 +15,48 @@ tags:
 
 {% endAside %}
 
-This page lists major known issues that affect developers as they migrate to Manifest V3. Known issues are divided into two primary groups:
+Recently, we announced changes to the Manifest V2 (MV2) deprecation timeline, and while we remain firmly committed to Manifest V3 (MV3) we acknowledge there is more work to do on our part including: 
 
-* **Capabilities** – Features that we plan to add to Manifest V3 to facilitate migration efforts.
-* **Bugs** – Significant issues with Manifest V3 platform features that are not working as expected.
+*   Before announcing a new timeline for deprecation, we will finish addressing prioritized platform gaps and closing critical bugs that are documented on this page.
+*   We will give developers time to build, guaranteeing at least 6 months between a timeline announcement and any experiments deprecating MV2 features.
```

Comment:
```suggestion
*   We will give developers time to build, guaranteeing at least 6 months between a timeline announcement and any experiments removing support for MV2.
```

I feel like this is a bit more accurate (we may _separately_ deprecate features as part of normal platform maintenance)